### DISCUSSION TITLE
My Final Semester in BusTub

### DISCUSSION THREAD

**Discussion by skyzh** [skyzh/skyzh-site#26] [General]
Link to the blog post: https://www.skyzh.dev/blog/2023-12-20-the-final-semester-in-bustub/

**Comment by benyucong** [comment]
Thank you so much for all the wonderful work! 
Best wishes for all your future projects!

**Comment by andrew-abo** [comment]
A belated thanks, Alex.  I really enjoyed completing all the Fall 2023 projects including project4 107/100 points.  It's definitely cool you brought SQL to Bustub, which makes it feel more realistic for us.  I had lots of fun discovering and fixing deadlocks in Project 4.
